--- muscleframework-1.1.3/XCardII/src/xcardPersonalizeData.cpp.orig	Thu Sep 26 20:39:11 2002
+++ muscleframework-1.1.3/XCardII/src/xcardPersonalizeData.cpp	Sat Mar 15 11:17:19 2003
@@ -40,7 +40,7 @@
     {
         QFont font("helvetica", 16, 50, FALSE);
         font.setStyleHint((QFont::StyleHint)5);
-        font.setCharSet((QFont::CharSet)0);
+//        font.setCharSet((QFont::CharSet)0);
         qtarch_Label_7->setFont( font );
     }
     qtarch_Label_7->setFocusPolicy(QWidget::NoFocus);
@@ -217,10 +217,10 @@
     selector_numTries->setFocusPolicy(QWidget::StrongFocus);
     selector_numTries->setBackgroundMode(QWidget::PaletteBackground);
     selector_numTries->setBackgroundOrigin( QWidget::WidgetOrigin );
-    selector_numTries->setFrameStyle( 51 );
-    selector_numTries->setLineWidth( 2 );
-    selector_numTries->setMidLineWidth( 0 );
-    selector_numTries->setMargin( 0 );
+//  selector_numTries->setFrameStyle( 51 );
+//  selector_numTries->setLineWidth( 2 );
+//  selector_numTries->setMidLineWidth( 0 );
+//  selector_numTries->setMargin( 0 );
     selector_numTries->setRange( 0, 99 );
     selector_numTries->setSteps( 1, 0 );
     selector_numTries->setPrefix( "" );
